app/lib/tufts/importer.rb

```diff
@@ -152,17 +150,34 @@ def message
       # Create a new Importer::Error for each error found by the Nokogiri parser
       def check_for_well_formed_xml(file)
         doc = Nokogiri::XML file
-        doc.errors.each do |e|
-          errors << Importer::Error.new(e.line, type: :serious, message: e.message)
+        if doc.errors.count > 0
+          e = doc.errors.first
+          errors << Importer::Error.new(e.line, type: :serious, message: "Malformed XML error: #{e.message}")
+        end
+        check_for_required_fields(doc)
+      end
+
+      # Given a record, check that it has all required fields
+      def check_for_required_fields(doc)
+        doc.root.add_namespace("dc", "http://purl.org/dc/terms/")
```

Comment:
Can you say something about the need to add the namespaces explicitly? Would it be better to pull the namespaces from the MiraXmlMapping?